src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1SpaceBookingEntity.kt

```diff
@@ -166,6 +226,16 @@ interface Cas1SpaceBookingRepository : JpaRepository<Cas1SpaceBookingEntity, UUI
   fun updateEventNumber(applicationId: UUID, eventNumber: String)
 }
 
+interface Cas1SpaceBookingDaySummarySearchResult {
+  val id: UUID
+  val crn: String
+  val canonicalArrivalDate: LocalDate
+  val canonicalDepartureDate: LocalDate
+  val tier: String
+  val releaseType: String
+  val characteristicsPropertyNames: String
```

Comment:
Changed from List<String> to String. 
Array_Agg was returning a list of char after rebase.  Couldn't see any direct reason. 
String_Agg is now being used.